src/components/Accordian/index.tsx

```diff
@@ -1,18 +1,18 @@
-import React, { FC, useState, useEffect } from 'react';
-import ReactHtmlParser from 'react-html-parser';
+import React, { FC, useState, useEffect } from "react";
+import ReactHtmlParser from "react-html-parser";
 
-import './Accordian.scss';
+import "./styles.scss";
 
 type AccordianProps = { data: Array<{ title: string; content: string }> };
-type AccordianItemState = { title: string; content: string; open: Boolean };
-type AccordianState = Array<{ title: string; content: string; open: Boolean }>;
+type AccordianItemState = { title: string; content: string; open: boolean };
+type AccordianState = Array<{ title: string; content: string; open: boolean }>;
 
 const Accordian: FC<AccordianProps> = ({ data }) => {
     const [accordionItems, setAccodionItems] = useState<AccordianState>([]);
 
     useEffect(() => {
         const parseData: () => void = (): void => {
-            let accordion: AccordianState = [];
+            const accordion: AccordianState = [];
 
             data.forEach((i: { title: string; content: string }) => {
```

Comment:
why not map instead of foreach

Reply:
cause I have not practiced using the array prototypes as much and been sticking to base stuff